org.mixedrealitytoolkit.uxcore/Editor/Inspectors/PressableButton/PressableButtonInspector.cs

```diff
@@ -101,7 +101,6 @@ private void OnSceneGUI()
                 return;
             }
 
-            serializedObject.Update();
```

Comment:
For future reference on removing this:

This line was originally added here:  https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11296, for the issue https://github.com/microsoft/MixedRealityToolkit-Unity/issues/11295.  This issue concered the PressableButton push planes not visually updating when changed manually in the Inspector.  I tested that after removing this line and it seems fine currently.  This makes sense as `serializedObject.ApplyModifiedProperties();` is called at the end of `PressableButton.DrawProperties()`, which should apply any Inspector changes to the target object used in `OnSceneGUI()`.